app/assets/javascripts/privileges.js

```diff
@@ -27,7 +27,10 @@ $(() => {
     const $input = $td.find('.js-privilege-edit');
     const name = $input.data('name');
     const type = $input.data('type');
-    const value = parseFloat($input.val() || '') || null;
+
+    const rawValue = parseFloat($input.val() || '')
```

Comment:
If `$input.val()` is `null` or `undefined` then the result will be `parseFloat('')` which is `NaN`. This works perfectly with the next line, but may not be obvious to all readers. Is it worth adding a hint that the result will be `NaN`?

I would expect the following change to still give identical results in all cases:

```suggestion
    const rawValue = parseFloat($input.val() || 'NaN')
```

Do you think this is more or less readable?

Reply:
I think it is more confusing :) A couple of years from now no one will understand why we short-circuit to a string `"NaN"`. A comment should suffice for that, methinks, a moment.

Reply:
One thing I just realized is that I am not even sure we need a short-circuit there as `parseFloat` will do the job either way.

Reply:
Good point about not needing the short circuit `||`. I should have checked what parseFloat does with `null` and `undefined` before commenting.

In that case, could we remove the `|| ''`?